Tests/ProductListDisplayTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+require_once 'src/Services/ProductListDisplayService.php';
+use PHPUnit\Framework\TestCase;
+
+class ProductListDisplayTest extends TestCase
```

Comment:
test classes should have *exactly* the same name as the class they test, plus the word `Test` on the end